packages/state-transition/package.json

```diff
@@ -59,7 +59,6 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@chainsafe/as-sha256": "^0.5.0",
-    "@chainsafe/bls": "7.1.3",
```

Comment:
Looks like it was added due to incorrect rebase, since https://github.com/ChainSafe/lodestar/pull/6894 we use blst directly in state-transition which seems questionable though, don't we want state-transition to be browser compatible?

Reply:
> don't we want state-transition to be browser compatible?

not currently